Packages/MIES/MIES_NeuroDataWithoutBorders.ipf

```diff
@@ -1190,13 +1188,17 @@ threadsafe static Function NWB_AppendSweepLowLevel(STRUCT NWBAsyncParameters &s)
 				params.channelSuffixDesc = NWB_SOURCE_TTL_BIT
 				params.stimset       = ttlStimsets[log(ttlBit)/log(2)]
 				NWB_GetTimeSeriesProperties(s.nwbVersion, s.numericalKeys, s.numericalValues, params, tsp)
-				params.groupIndex    = IsFinite(params.groupIndex) ? params.groupIndex : GetNextFreeGroupIndex(s.locationID, path)
+				params.groupIndex      = IsFinite(params.groupIndex) ? params.groupIndex : GetNextFreeGroupIndex(s.locationID, path)
+				WAVE/T/Z params.epochs = EP_FetchEpochs(s.numericalValues, s.textualValues, s.sweep, log(ttlBit)/log(2), params.channelType)
```

Comment:
From readability I think it would be better to put the "log(ttlBit)/log(2)" calc in a variable, it is also used at least twice.
This would also allow for a descriptive name.

Reply:
Fix that only in the HEAD version of this PR. Introduce a common function which also does proper checks for integers, maybe find a solution without having to use FP functions like log.

Reply:
Done.